core/src/components/refresher/readme.md

```diff
@@ -154,7 +154,6 @@ export const RefresherExample: React.FC = () => (
     </IonContent>
   </IonContent>
 );
```

Comment:
Shouldn't there be an Android usage too?

https://github.com/ionic-team/ionic/blob/4282be430ab898ee3f135356d24868c181276e6d/core/src/components/refresher/readme.md#native-refreshers

Reply:
somehow I deleted my Android docs 🤔 will fix